### ISSUE TITLE
SIGABRT on page load

### ISSUE THREAD

**Issue by rimio** [atlas-engineer/nyxt#1862]
**Describe the bug**
When (o)pening any URL, upon CR after typing the address the browser silently NOOPs, and upon the second CR it fails with SIGABRT.

**Precise recipe to reproduce the issue**
Open any web page.

For website-specific issues:
Can you reproduce this issue with Epiphany / GNOME Web (https://wiki.gnome.org/Apps/Web)? No, obviously.

**Information**
- OS name+version: ArchLinux, up to date
- Graphics card and driver: NVIDIA 2070, `nvidia-495.44-3`
- Desktop environment / Window manager name+version: i3-gaps
- How you installed Nyxt (Guix pack, package manager, build from source): Compiled from AUR (https://aur.archlinux.org/packages/nyxt/)
- Information from `show-system-information`:
```
Nyxt version: 2.2.2
Renderer version: GI-GTK
Operating system kernel: Linux 5.14.16-arch1-1
Lisp implementation: SBCL 2.1.9 (Dynamic space size: 1073741824)
Features: (:WEBKIT2 :WEBKIT2-2.34 :WEBKIT2-PASTE-PLAINTEXT :WEBKIT2-TRACKING
 :WEBKIT2-MUTE :WEBKIT2-EMOJI :WEBKIT2-MEDIA :WEBKIT2-SANDBOXING :GTK-3-22
 :GTK-3-20 :GTK-3-18 :GTK-3-16 :GTK-3-14 :GTK-3-12 :GTK-3-10 :GTK-3-8 :GTK-3-6
 :GTK-3-4 :GTK :GDK-3-22 :GDK-3-20 :GDK-3-18 :GDK-3-16 :GDK-3-14 :GDK-3-12
 :GDK-3-10 :GDK-3-8 :GDK-3-6 :GDK-3-4 :CAIRO-1-10 :CAIRO-1-12 :GDK-PIXBUF
 :GLIB-2-30 :GLIB-2-32 :GLIB-2-34 :GLIB-2-36 :GLIB-2-38 :GLIB-2-40 :GLIB-2-42
 :GLIB-2-44 :GLIB-2-46 :GLIB-2-48 :GLIB-2-50 :GLIB-2-52 :GLIB-2-54 :GLIB-2-56
 :GLIB-2-58 :GLIB :NYXT-2 :FSET-EXT-STRINGS :CUSTOM-HASH-TABLE-NATIVE :SWANK
 :PLUMP-UTF-32 :GLOBAL-VARS :DECLARE-TYPES :PARENSCRIPT
 :SBCL+SAFE-STANDARD-READTABLE :NAMED-READTABLES :LPARALLEL :21BIT-CHARS
 :CHUNGA :CLOSER-MOP :CL-PPCRE-UNICODE :FLEXI-STREAMS :CL-UNICODE :CL-PPCRE
 :CL-JSON-DOUBLE-FLOAT-IS-SUBSUMED :CL-JSON-SINGLE-FLOAT-IS-SUBSUMED
 :BORDEAUX-THREADS :LPARALLEL.WITH-CLTL2 :LPARALLEL.WITH-CAS
 :LPARALLEL.WITH-STEALING-SCHEDULER :SPLIT-SEQUENCE CHIPZ-SYSTEM:GRAY-STREAMS
 CFFI-FEATURES:FLAT-NAMESPACE CFFI-FEATURES:X86-64 CFFI-FEATURES:UNIX :CFFI
 CFFI-SYS::FLAT-NAMESPACE ALEXANDRIA::SEQUENCE-EMPTYP :FAST-IO-SV :FAST-IO
 :SBCL-USES-SB-ROTATE-BYTE :CL-JSON-CLOS :CL-JSON :THREAD-SUPPORT :ASDF3.3
 :ASDF3.2 :ASDF3.1 :ASDF3 :ASDF2 :ASDF :OS-UNIX :NON-BASE-CHARS-EXIST-P
 :ASDF-UNICODE :X86-64 :GENCGC :64-BIT :ANSI-CL :COMMON-LISP :ELF
 :IEEE-FLOATING-POINT :LINUX :LITTLE-ENDIAN :PACKAGE-LOCAL-NICKNAMES
 :SB-CORE-COMPRESSION :SB-LDB :SB-PACKAGE-LOCKS :SB-THREAD :SB-UNICODE :SBCL
 :UNIX)

ASDF version: 3.3.1
ASDF registries: (NYXT-SOURCE-REGISTRY ENVIRONMENT-SOURCE-REGISTRY)
Critical dependencies: (/home/rimio/aur/nyxt/src/nyxt/_build/cl-cffi-gtk/gtk/cl-cffi-gtk.asd
 /home/rimio/aur/nyxt/src/nyxt/_build/cl-gobject-introspection/cl-gobject-introspection.asd
 /home/rimio/aur/nyxt/src/nyxt/_build/cl-webkit/webkit2/cl-webkit2.asd)
```

**Output when started from a shell**
```
$ nyxt
<INFO> [19:18:13] Listening to socket "/run/user/1000/nyxt/nyxt.socket".
Nyxt version 2.2.2
<INFO> [19:18:13] Loading Lisp file "/home/rimio/.config/nyxt/auto-config.lisp".

** (nyxt:92949): ERROR **: 19:18:19.845: GApplication is required for xdg-desktop-portal access in the WebKit sandbox.
<WARN> [19:18:19] Warning: Error on GTK thread: The value
                       NIL
                     is not of type
                       SB-KERNEL:CODE-COMPONENT

** (process:92949): ERROR (recursed) **: GApplication is required for xdg-desktop-portal access in the WebKit sandbox.fatal error encountered in SBCL pid 92949 tid 92966:
SIGABRT received.

   0: fp=0x7fa0f31c28f0 pc=0x7fa0f303cd22 Foreign function gsignal
```

**Comment by Ambrevar** [comment]
I believe this is a duplicate of #1840.

**Comment by Ambrevar** [comment]
All WebKitGTK 2.34-related hangs / crashes should be fixed on master since c2f1e61.

We had to stop enforcing the sandboxing though. We are tackling this in a separate issue, see #1781.
